internal/client.go

```diff
@@ -794,6 +798,13 @@ type (
 		onConflictOptions *OnConflictOptions
 	}
 
+	// startWorkflowResultInfo can be passed to StartWorkflowOptions to receive additional information
+	// of StartWorkflowExecution response.
+	startWorkflowResultInfo struct {
```

Comment:
While you are here, would like two more booleans: `Started` and `StartedEagerly`. `Started` is very helpful for users that do use-existing, update with start, and those not setting `WorkflowExecutionErrorWhenAlreadyStarted`. `StartedEagerly` is something users may want to know, and in general our tests would like to know this too. If we want to do this in a separate issue, no prob.

Reply:
Seems unnecessary in this PR since these fields won't be accessible to users anyway

Reply:
:+1: I didn't realize it is intentionally unexposed.